src/sdgio-edit.owl

```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020246> "2019-04-29T16:29:41Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020246> <https://clarisa.cgiar.org/api/srf-idos/9>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020246> "enhanced benefits from ecosystem goods and services"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020246>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020247> (more sustainably managed agro-ecosystems)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020247> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020247> "2019-04-29T16:30:19Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020247> <https://clarisa.cgiar.org/api/srf-idos/10>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020247> "more sustainably managed agro-ecosystems"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020247>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020248> (mitigation and adaptation achieved)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020248> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020248> "2019-04-29T16:31:05Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020248> <https://clarisa.cgiar.org/api/srf-idos/A1>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020248> "mitigation and adaptation achieved"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020248>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020249> (equity and inclusion achieved)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020249> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020249> "2019-04-29T16:31:30Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020249> <https://clarisa.cgiar.org/api/srf-idos/B1>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020249> "equity and inclusion achieved"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020249>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020250> (enabling environment improved)
```

Comment:
We need a class in SDGIO for "enabling environment" - which I suspect will be difficult to define.

Reply:
@pbuttigieg Shouldn't at least the instance be defined as an instance of the IDO class?

Reply:
@marieALaporte Yes, there should be a "Types" assertion that this is an IDO.

@Mttbnchtt your protege should look like this:

![image](https://user-images.githubusercontent.com/2276146/56959005-55a3de80-6b4c-11e9-93ce-f79258d23f18.png)

Reply:
@marieALaporte noted that the instance SDGIO_00020250 did not have. I added the instance to the class "intermediate development outcome".

@pbuttigieg: the class that you showed in the picture looks good, I suppose. Is there a problem with that one?

Reply:
@Mttbnchtt everything looks good. Thanks.

Reply:
> @pbuttigieg: the class that you showed in the picture looks good, I suppose. Is there a problem with that one?

I added that quickly for visualisation but didn't commit it. You've fixed this now.

Reply:
@pbuttigieg: Do I need to define a class "enabling environment" now or should I rather create an issue?